tools/vagrant-box-atom/upload.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/ruby
+
+require 'bundler/setup'
+require 'net/http'
+require 'vagrant_cloud'
+
+ORG = 'artefactual'
+BOX = 'atom'
```

Comment:
At the moment it's all named for AtoM's Vagrant box, but we could accept this as a script argument and reuse the same script for Archivematica's Vagrant box, if we want to implement a similar workflow.